Cython/Compiler/UtilNodes.py

```diff
@@ -147,6 +147,9 @@ def update_expression(self, expression):
         if type:
             self.type = type
 
+    def analyse_target_declaration(self, env):
+        return  # because it can be the left-hand sides
```

Comment:
I don't understand this comment. What would happen without the `return`?

Reply:
I think what the comment referred to the need for the function rather than the need for the `return`. I'll change it to make it clearer